### ISSUE TITLE
Highcharts: Sunburst chart series data labels outside the leaf level nodes with connectors

### ISSUE THREAD

**Issue by amruta30** [highcharts/highcharts#7379]
#### Expected behaviour
Refer the Sunburst chart's series data labels on the following link:-
http://www.dundas.com/support/blog/sunburst-charts-homerun-or-groundout

Expected behaviour is to configure the series data labels in such a way that if the arc size of a node is less than a particular value, instead of hiding the data label, show it outside the leaf level nodes of the chart, also adjust the distance of data labels and its connectors from the chart.
Is there any way to achieve this in Highcharts Sunburst chart?

#### Actual behaviour
Refer the following JS Fiddle:
http://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/sunburst
It is observed that all series data labels when shown, overlap or only the show/hide behaviour of data labels can be achieved through the formatter function. Its observed that connectors for series data labels of leaf level nodes or the position of series data labels outside are not configurable.

#### Live demo with steps to reproduce
http://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/sunburst

**Comment by pawelfus** [comment]
Hi @amruta30 

Thank you for sharing your idea. At this moment we can not change where datalabels are rendered on the same level.

**Comment by TorsteinHonsi** [comment]
For the records @amruta30, this demo actually (accidentally) uses a not-yet released feature, the `dataLabels.filter`. See http://jsfiddle.net/highcharts/kkx12u6x/ for how it will look with the new feature.

**Comment by amruta30** [comment]
@TorsteinHonsi I checked your JS Fiddle: http://jsfiddle.net/highcharts/kkx12u6x/ . Instead of hiding the series data labels, its expected to show the data labels outside the leaf level nodes and connectors to them.

**Comment by TorsteinHonsi** [comment]
Yes, I appreciate that. So we added the _Enhancement_ label to this issue.

**Comment by abishek-musashi** [comment]
Hello, Is this enhancement added to your release? if so when and how can I add this to my react library
